code/src/index.css

```diff
@@ -5,9 +5,62 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: url(./asstes/pexels-eugene-liashchevskyi-10843173.jpg);
+  background-size: cover; 
+  background-position: 50%;
+  font-family: 'Poppins', sans-serif;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.complete-todo {
+  text-decoration: line-through;
+  margin: 0;
+}
+
+.uncomplete-todo {
+  margin-top: -10px;
+  margin: 0;
+}
+
+/* Scrollbar styling  */
+::-webkit-scrollbar {
+  width: 5px;
+}
+
+::-webkit-scrollbar-track {
+  background: #f1f1f1;
+}
+
+::-webkit-scrollbar-thumb,
+::-webkit-scrollbar-thumb:hover {
+  background: #888;
+}
+
+.scrollbar-hidden::-webkit-scrollbar {
+  display: none;
+}
+
+/* Hide scrollbar for IE, Edge add Firefox */
```

Comment:
Thanks for the comment! Which scrollbar? Sorry if I don't understand haha.. I tried looking in my Firefox browser